tests/queries/0_stateless/01834_alias_columns_laziness_filimonov.sh

```diff
@@ -12,11 +12,7 @@ insert into aliases_lazyness(x) select * from numbers(100);
 
 # In very old ClickHouse versions alias column was calculated for every row.
 # If it works this way, the query will take at least 0.1 * 100 = 10 seconds.
-# If the issue does not exist, the query should call sleepEachRow() "only" 4 times:
-# - from MergeTreeData::getQueryProcessingStageWithAggregateProjection() -> MergeTreeWhereOptimizer -> getBlockWithConstants()
-# - from MergeTreeWhereOptimizer -> getBlockWithConstants()
-# - ReadFromMergeTree::selectRangesToRead() -> getBlockWithConstants()
-# - Pipeline
```

Comment:
Now it behaves in more expectable way. `sleepEachRow()` forbids constant folding and thus it's not executed by `ExpressionAnalyzer`, now it's executed only by pipeline.